opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/google/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.11.dev0"
+__version__ = "0.12.dev0"
```

Comment:
Shouldn't this be 0.11?

Reply:
the dev version is `0.12.dev0` (after the 2nd commit), the first commit does `0.11.b0` which is the right version for the release.